.scalafmt.conf

```diff
@@ -0,0 +1,2 @@
+maxColumn = 100
```

Comment:
I just checked in my editor windows - 100 chars per line seems a little narrow. I would opt for 120 or even 130.